transport_service_queries/src/main/java/pl/edu/pg/transport/GetFlightDetailsEvent.java

```diff
@@ -0,0 +1,14 @@
+package pl.edu.pg.transport;
+
+public class GetFlightDetailsEvent {
```

Comment:
What do you think about creating the base class, for example`GenericEvent`, which will contain common part for all types of events?

Reply:
Or we can pass the source as a header, wdyt?

Reply:
`GenericEvent` sounds great. There was a discussion about headers (https://stackoverflow.com/questions/42593804/rabbitmq-using-custom-headers-to-store-message-parameters) and it seems that `source` should be in the body of messages.